packages/react-native-windows-init/src/Cli.ts

```diff
@@ -124,6 +124,13 @@ export const windowsInitOptions = initOptions({
     default: undefined, // This must be undefined because we define the conflicts field below. Defining a default here will break the version option
     conflicts: 'version',
   },
+  template: {
+    type: 'string',
+    describe: '[Experimental] Specify a project template to use',
+    hidden: true,
+    default: undefined,
```

Comment:
This has to be undefined because it has a conflicts field, but we should comment that here.